### ISSUE TITLE
JavaScript to CSS converter online?

### ISSUE THREAD

**Issue by KielYam1212008** [ritz078/transform#325]
I do not know how to convert javascript to css in the website tools online.

**Comment by lil5** [comment]
Shameless plug a project to move from JSS to SCSS
https://lil5.github.io/jss-to-scss/

It's not perfect but it does the job
 
---

For the devs @ritz078 Here is the format function I use (a list of regexs) are you interested in this conversion tool?

https://github.com/lil5/jss-to-scss/blob/main/src/jss-to-scss/index.ts

**Comment by RIP21** [comment]
Yeah, having a such thing that understands typical chakra/mantine/theme-ui more or less (with these multiple shortcut property names like `h` `w`, `mx`, `my`, `pl` etc., and outputs CSS would be awesome. (then I'll drop that to Tailwind converter and I'm gold :D)

I assume `emotion` itself or any CSS-in-JS library that uses all these helpers can output these if needed.
